be/src/Project/Web/Seller/Seller.Web/Seller.Web.csproj

```diff
@@ -43,4 +43,7 @@
 		<Folder Include="Areas\Inventory\Views\Warehouse\" />
 		<Folder Include="wwwroot\" />
 	</ItemGroup>
+	<ItemGroup>
```

Comment:
@dawiddworak88 

I wanted to use the implementation in ApiFixture for this, but I can't use it for two projects. I tried to give an alias, but there is no such property